src/libertem/web/cli.py

```diff
@@ -13,16 +14,19 @@
               help='enable/disable opening the browser', default='True')
 @click.option('-l', '--log-level', help=f"set logging level. Default is 'info'. {log_values}",
               default='INFO')
+@click.option('--instance_type', type=click.Choice(["liberated", "restricted"]), default="liberated")
+@click.option('--instance_config', type=str)
```

Comment:
Currently, the instance_config contains the details on how many workers and cuda devices we need to initiate the server with. This is to replace the connection form that LiberTEM currently offers when the user first connects.